src/panels/config/integrations/integration-panels/ozw/ozw-node-dashboard.ts

```diff
@@ -87,20 +87,31 @@ class OZWNodeDashboard extends LitElement {
           ${this._node
             ? html`
                 <ha-card class="content">
-                  <div class="card-content">
-                    <b
-                      >${this._node.node_manufacturer_name}
-                      ${this._node.node_product_name}</b
-                    ><br />
-                    Node ID: ${this._node.node_id}<br />
-                    Query Stage: ${this._node.node_query_stage}
-                    ${this._metadata?.metadata.ProductManualURL
-                      ? html` <a
-                          href="${this._metadata.metadata.ProductManualURL}"
-                        >
-                          <p>Product Manual</p>
-                        </a>`
-                      : ``}
+                  <div class="card-content flex">
+                    <div class="node-details">
+                      <b>
+                        ${this._node.node_manufacturer_name}
+                        ${this._node.node_product_name}
+                      </b>
+                      <br />
+                      Node ID: ${this._node.node_id}<br />
+                      Query Stage: ${this._node.node_query_stage}
+                      ${this._metadata?.metadata.ProductManualURL
+                        ? html` <a
+                            href="${this._metadata.metadata.ProductManualURL}"
+                          >
+                            <p>Product Manual</p>
+                          </a>`
+                        : ``}
+                    </div>
+                    <div class="product-image">
+                      ${this._metadata?.metadata.ProductPicBase64
```

Comment:
Can we btw do this differently? Sending large files like images over the WebSocket is not recommended, would rather have just the URL or something.

Reply:
OZW just has the base64 encoded image in the MQTT data for the node. Though I suppose we could register a regular HTTP endpoint on the backend to return the image instead of feeding it over the websocket. I've never done anything with custom endpoints like that in the backend though.

Reply:
Something for a future PR maybe